doc/designs/ddd/ddd-01-conn-blocking.c

```diff
@@ -141,15 +141,15 @@ int main(int argc, char **argv)
     SSL_CTX *ctx = NULL;
     BIO *b = NULL;
     char buf[2048];
-    int l, res = 1;
+    int l, mlen, res = 1;
 
     if (argc < 3) {
         fprintf(stderr, "usage: %s host port\n", argv[0]);
         goto fail;
     }
 
-    snprintf(host_port, sizeof(host_port), "%s:%s\n", argv[1], argv[2]);
-    snprintf(msg, sizeof(msg),
+    snprintf(host_port, sizeof(host_port), "%s:%s", argv[1], argv[2]);
```

Comment:
the stray newline character has been removed.

when it was present, the demo did not work at all for me.